data/pug/pug outfits.txt

```diff
@@ -102,7 +102,7 @@ outfit "Pug Seeker"
 		"shield damage" 24
 		"hull damage" 13
 		"hit force" 3
-		"missile strength" 40
+		"missile strength" 10
```

Comment:
I like the idea of lowering the strength. However, 10 feels... a little low.
With these changes, very few of the Seekers actually hit their targets, and one type of Pug ship is dedicated to only Seekers. The Enfolta will be useless in this case.

So perhaps bump this up to 15 or so, just so that there are a few uses for some ships. (of course, if you've already tested FW battles with this and it is possible to lose sometimes, ignore)

Also, does this affect the Pug fighting the Unfettered anywhere? I haven't yet finished the Wanderer storyline, so I don't know if there are big Pug-UHai fights.

Reply:
In the first pug fight in reconciliation, there are a total of 20 Pug Seekers deployed in the battle, and a total of 10 Heavy anti-missiles. The total number of seekers shot down, therefore, will be at most 1/3 (and potentially fewer if the pug don't evenly spread their fire). The player might bring a few more, but unless they're running a heavy dedicated AM build you're looking at more than half of the seekers going through.
In the first fight in Checkmate, there are a total of 46 Pug Seekers deployed, and a total of 14 heavy and 3 regular AMs - hence, an even smaller portion of the seekers will be shot down, and the same amount of AM brought by the player will be an even smaller contribution.

So, I don't think it's at all accurate to say that "very few of the Seekers actually hit their targets" with the changes.

Also, the Enfolta has two seekers, two zappers and a zapper turret, so I don't think it's accurate to say that it's "dedicated to only Seekers".

Reply:
40 to 10 is a big drop, but that's because 40 was such an absurd value to begin with. 40 is the same missile strength as the Typhoon, which fires 12 times slower with less than half the projectile velocity.